toolkit/scripts/prerequisites.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+set -e
+
+prereq_md_path=""
+prereq_json_path=""
+
+function print_help {
+    echo "Pre-requisite installer. Installs azurelinux pre-requisites for the toolkit."
```

Comment:
Is this installing the pre-requisites? It seems like this script has 2 jobs: Listing the pre-reqs and checking the Markdown file.

Reply:
Ah, didn't update that when I reworked the script  a bit. I'll add some extra details to the help.